utils/types.go

```diff
@@ -79,6 +79,13 @@ type KongClientConfig struct {
 	Headers []string
 }
 
+// ForWorkspace returns a copy of KongClientConfig that produces a KongClient for the workspace specified by argument.
+func (kc *KongClientConfig) ForWorkspace(name string) KongClientConfig {
```

Comment:
Nice one. I like this.

Reply:
Realized that I missed the `ping` operation. Fixed in e61a5b7.